subprojects/core/src/main/java/org/gradle/api/tasks/AbstractCopyTask.java

```diff
@@ -512,6 +519,15 @@ public AbstractCopyTask expand(Map<String, ?> properties) {
         return this;
     }
 
+    /**
+     * {@inheritDoc}
+     */
+    @Override
+    public AbstractCopyTask expand(Map<String, ?> properties, Action<? super ExpandDetails> action) {
```

Comment:
@wolfs `ExpandDetails.escapeBackslash` impacts the output of this task. Is there some place where we track this as inputs or does this fall into the bucket of things we're not tracking properly?

Reply:
We are not tracking the expansion at all: https://github.com/gradle/gradle/issues/1191